nncf/torch/nncf_network.py

```diff
@@ -440,6 +480,22 @@ def strip_fn(tensor: TracedTensor) -> torch.Tensor:
         kwargs = objwalk(kwargs, is_traced_tensor_predicate, strip_fn)
         return args, kwargs
 
+    def create_knowledge_distillation_loss_handler(self, kd_original_model, calculate_fn):
+        """
+        Creates KnowledgeDistillationLossHandler instance for enabling Knowledge Distillation feature.
+            Also returns created KnowledgeDistillationLossHandler for control over Knowledge Distillation logic.
+
```

Comment:
Add same note about the KD loss handler only representing the KD loss corresponding to the inputs of the latest `NNCFNetwork`'s forward call.